### ISSUE TITLE
Gopass master fails to load on windows

### ISSUE THREAD

**Issue by martinhoefling** [gopasspw/gopass#1539]
### Summary

Gopass master build (cf6ebc6d57c5b54185040942d84d1d9a78a20eec) fails with

```
$ ./gopass.exe

Error: Failed to initialize store: failed to initialized stores: failed to initialize the root store at '/C:\Users\Martin\.password-store': failed to init storage backend: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: failed to init storage backend: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: failed to initialize the root store at '/C:\Users\Martin\.password-store': failed to init storage backend: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: failed to init storage backend: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.: unknown storage backend: mkdir /C:: The filename, directory name, or volume label syntax is incorrect.
```

Debug log attached.
[log.txt](https://github.com/gopasspw/gopass/files/5092576/log.txt)

**Comment by dominikschulz** [comment]
```
Loaded config from C:\Users\Martin\.config\gopass\config.yml: &config.Config{AutoClip:true, AutoImport:true, ClipTimeout:45, ExportKeys:false, NoColor:false, NoPager:false, Notifications:true, Path:"/C:\\Users\\Martin\\.password-store", SafeContent:false, Mounts:map[string]string{}, configPath:"C:\\Users\\Martin\\.config\\gopass\\config.yml", XXX:map[string]interface {}(nil)}
```

Looks like your `Path` value is incorrect: ` Path:"/C:\\Users\\Martin\\.password-store"`.

Could try with either:

* No config
* Fixing the Path, i.e. removing the leading `/`?

**Comment by zeysh** [comment]
Tested with "no config" and it works. 
Thanks @dominikschulz

**Comment by AnomalRoil** [comment]
I think this is now fixed, and Windows build are working since #1680 and #1632
